.github/workflows/test.yml

```diff
@@ -149,6 +149,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           CTN_BUILDER_NAME: ${{ steps.builder.outputs.name }}
+          TEST_FOR_SUMMARY: ${{ secrets.TEST_FOR_SUMMARY }}
```

Comment:
This dummy secret is used for testing the behavior of a user having repo name as secret value.